pwiz_tools/Skyline/Test/SchemaDocumentsTest.cs

```diff
@@ -73,11 +73,10 @@ public void TestDocumentFormatSchemaFiles()
             var xsdCurrentResourceName = SchemaDocuments.GetSkylineSchemaResourceName(DocumentFormat.CURRENT.ToString()); // Special case, returns "Skyline_Current.xsd"
             var currentXsdContents = GetResourceText(xsdCurrentResourceName); 
             Assert.IsNotNull(currentXsdContents);
-            if (explicitCurrentXsdContents != null)
+            if (explicitCurrentXsdContents != null && !explicitCurrentXsdContents.Equals(currentXsdContents))
             {
-                Assert.AreEqual(currentXsdContents, explicitCurrentXsdContents, 
-                    string.Format("Expected resource files {0} and {1} to be identical - did you change the contents of Skyline_Current.xsd without incrementing the value of DocumentFormat.CURRENT?",
-                        explicitCurrentResourceName, xsdCurrentResourceName));
+                AssertEx.Fail(string.Format("Expected resource files {0} and {1} to be identical - did you change the contents of Skyline_Current.xsd without incrementing the value of DocumentFormat.CURRENT?",
```

Comment:
We could also use AssertEx.NoDiff() which outputs the first lines that are different, and may offer more information, but still not blast the entire file contents into the log.

Reply:
Yeah, that would be better - will do.